### ISSUE TITLE
Should MB02 and MB04 use fixed photo year or OS_YEAR column

### ISSUE THREAD

**Issue by edwardsmarc** [edwardsmarc/CASFRI#581]
CAS04 used fixed year values of 1998 for MB02 and 2000 for MB04.

There is a nos_year column but the values range from 2000 to 2002 for MB02 and MB04.

**Comment by MelinaHoule** [comment]
The team received the inventory in december 2007 on a DVD. Documentations and a jpeg containing year of photography was provided with the data. 

As you said, there is a os_year column provided in the source data (ranging from 2000-2002), but the column is 98% empty. I can't tell whether we were told to use the JPEG to derive the photoyear or we decided to use it since they provided the photoyear jpeg with the data... I joined the project in 2009. 

I think because the photoyear were maximum 4 years appart, probably the decision to apply 1998 and 2000 to every polygons would allow us to get photoyear for all polygons with an error ranging from 0-4 years.
